### PR TITLE
[core] Language Right to Left

### PR DESCRIPTION
**Pull request by mart-e**
Followup of  #19892

Task:https://www.odoo.com/web?#id=27574&view_type=form&model=project.task&action=333&active_id=131&menu_id=4720
Pad:https://pad.odoo.com/p/r.bb7042f57bc2ecc1806cab9350775c75

### CODE REVIEW

**Comment by mart-e** [comment]
@rop-odoo I rebased your master branch. Do you confirm it contained the same commits as the 11.0 version at #19892 ?

File: odoo/addons/base/static/src/css/modules.less [hunk 1]
```diff
@@ -2,21 +2,21 @@
 .oe_module_icon {
     width: 80px;
     height: 80px;
-    float: left;
+    .float(left);
```
Comment (mreficent) [inline odoo/addons/base/static/src/css/modules.less]: I think this file should go in a less folder

**Comment by Yenthe666** [comment]
@mart-e what happend to this big guy? What is the conclusion?

**Comment by mart-e** [comment]
@Yenthe666 another round is in progress, @msh-odoo is working on it!

**Pull request closed without merging**